### DISCUSSION TITLE
Debug Launch Configuration for "Debug Frontend: (...)" does not work on WSL

### DISCUSSION THREAD

**Discussion by Zaubeerer** [fastapi/full-stack-fastapi-template#1375] [Questions] [answered]
I installed chrome on WSL via `sudo apt install --fix-missing ./google-chrome-stable_current_amd64.deb`.

However I got the following error:

```plain
Unable to find chrome version *. Available auto-discovered versions are: []. You can set the "runtimeExecutable" in your launch.json to one of these, or provide an absolute path to the browser executable.
```

I then added `runtimeExecutable`, resulting in the following launch config:
```json
  {
      "type": "chrome",
      "request": "launch",
      "name": "Debug Frontend: Launch Chrome against http://localhost:5173",
      "url": "http://localhost:5173/*",
      "webRoot": "${workspaceFolder}/frontend",
      "runtimeExecutable": "/usr/bin/google-chrome"
  },
```

However, I still get the following error:

```sh
Error running browser: spawn /usr/bin/google-chrome ENOENT
```

Executing `google-chrome` from terminal works fine.

Did anyone already encounter and solve this?

I guess this is rather a VS Code / extension issue, any hints where to better create this bug report issue?

**Comment by tiangolo** [comment, marked as answer]
Yep, I'm afraid this would really belong in the VS Code repo, not here. :sweat_smile: I'll close this one one. :coffee:
